frontend/src/app/components/network-graph/network-graph.component.ts

```diff
@@ -99,7 +103,9 @@ export class NetworkGraphComponent implements OnInit, OnChanges {
   private nextKey = 0;
   private keyToDatum = {};
 
-  constructor(private config: NetMetricsConfig) {
+  private subscription: Subscription;
+
+  constructor(private config: NetMetricsConfig, private route: ActivatedRoute) {
```

Comment:
Wait how are constructor arguments used? Do they just get attached to the `this` context? :scream:

Reply:
Yes, that is just typescript :sunglasses: Using a accessibility modifier makes the parameter a member. See https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties

Reply:
Good to know. Cool feature I guess (hysterically thinking about if I accidentally exposes properties unintended :laughing: ).